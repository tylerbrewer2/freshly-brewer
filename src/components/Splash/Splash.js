import React, { Component } from 'react';
import './Splash.css';

export default class Splash extends Component {
  render() {
    return (
      <div className="Splash">
        <div className="SplashFooter">
          <h1 className="SplashHeader">Coming soon</h1>
        </div>
      </div>
    );
  }
}
