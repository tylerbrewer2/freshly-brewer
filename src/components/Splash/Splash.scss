@keyframes slow-zoom {
  from   { transform: scale(1); }
  to     { transform: scale(1.3); }
}

@keyframes fade {
  from { opacity: 0 }
  to   { opacity: 1 }
}

.Splash {
  width: 100%;
  height: 100%;
}

.Splash:before {
  content: '';
  position: absolute;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  background: url(../../images/ring.JPG) no-repeat center center fixed; 
  background-size: cover;
  animation: slow-zoom 60s;
}

.SplashFooter {
  position: fixed;
  bottom: 0px;
  width: 100%;
  height: 50px;
  background-color: rgba(0,0,0,.9);
  opacity: 0;
  /*forwards keeps the final state of the animation*/
  animation: fade 5s ease-in-out 1s forwards;
}

.SplashHeader {
  font-size: 20px;
  color: white;
  margin-top: 12px;
  text-align: center;
}
