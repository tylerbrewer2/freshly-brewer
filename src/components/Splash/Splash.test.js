import React from 'react';
import renderer from 'react-test-renderer';
import Splash from './Splash';

describe('component structure', () => {
  it('renders with the correct structure', () => {
    const tree = renderer.create(
      <Splash />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

