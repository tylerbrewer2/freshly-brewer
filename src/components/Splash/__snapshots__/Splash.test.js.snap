exports[`component structure renders with the correct structure 1`] = `
<div
  className="Splash">
  <div
    className="SplashFooter">
    <h1
      className="SplashHeader">
      Coming soon
    </h1>
  </div>
</div>
`;
